.github/workflows/on-prerelease.yml

```diff
@@ -41,13 +41,21 @@ jobs:
           Import-Module ./github-builder.psm1
           Get-VersionNumber
 
-      - name: Create NuGet Package
+      - name: Publish .NET Assemblies
         run: |
           $packageVersion = "${{ steps.versions.outputs.dms-semver }}"
 
-          Import-Module ./github-builder.psm1
+          ./build.ps1 -Command BuildAndPublish `
```

Comment:
@CSR2017 yesterday you asked me if you could update the GitHub build to use `build.ps` and I answered yes without looking at the details. I thought you were replacing `dotnet build` and `dotnet test` (for example) with the build script.

It turns out there are specific reasons that we use `github-builder.psm1` in the GitHub build process. We could probably improve by combining in some way. But right now, we need to revert the change above: `BuildAndPublish` does not create the proper version number for the assemblies and package. This is one of the tasks performed in `github-builder`. There are some other tasks handled there, which we do not need when running local build and test processes.